tests/test-rest-nav-menus-controller.php

```diff
@@ -6,12 +6,31 @@ class WP_Test_REST_Nav_Menus_Controller extends WP_Test_REST_Controller_Testcase
 	 */
 	public $menu_id;
 
+	/**
+	 * @var
```

Comment:
Missing types.

Reply:
Fixed in https://github.com/WP-API/menus-endpoints/pull/41/commits/565a06fff290cddae77b2167a2d05b457e85e1d6